test/Microsoft.Azure.Devices.Edge.Test/Module.cs

```diff
@@ -93,66 +91,6 @@ public async Task TempSensor()
             await sensor.WaitForEventsReceivedAsync(startTime, token);
         }
 
-        [Test]
-        [Category("CentOsSafe")]
-        public async Task ImageGarbageCollection()
```

Comment:
@nlcamp Did you forget to add the new file to the commit? This commit only removes the test from `Module.cs`.